### PR TITLE
Multiverse Revival (now with huds!)

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
:cl: Kor
add: Multiverse teams now have HUD icons.
add: Multiverse war is back in the spellbook.
/:cl:

### CODE REVIEW

File: code/game/gamemodes/gang/gang_datum.dm [hunk 1]
```diff
@@ -134,7 +139,8 @@
 /datum/gang/proc/income()
 	if(!bosses.len)
 		return
-
+	if(fighting_style == "multiverse")
```
Comment (optimumtact) [inline code/game/gamemodes/gang/gang_datum.dm]: IF YOU HAVE TO PUT STUFF LIKE THIS IN THE CODE, IT'S SNOWFLAKE CODE YOU'RE WRITING AND YOU NEED TO REFACTOR THE SYSTEMS TO BE GENERIC

Comment (Jalleo) [inline code/game/gamemodes/gang/gang_datum.dm]: What the fruity person said

Comment (KorPhaeron) [inline code/game/gamemodes/gang/gang_datum.dm]: aaaaaaaaaa

Comment (KorPhaeron) [inline code/game/gamemodes/gang/gang_datum.dm]: wow I just thought of an obvious not garbage way to do this

File: code/game/gamemodes/gang/gang.dm [hunk 2]
```diff
@@ -193,6 +193,8 @@ var/list/gang_colors_pool = list("red","orange","yellow","green","blue","purple"
 	var/removed
 
 	for(var/datum/gang/G in gangs)
+		if(G.fighting_style == "multiverse" && !remove_bosses)
```
Comment (KorPhaeron) [inline code/game/gamemodes/gang/gang.dm]: this is still bad

Comment (KorPhaeron) [inline code/game/gamemodes/gang/gang.dm]: or I can just remove this and let loyalty implants deconvert multiverse people but that'd be sorta weird

Comment (GunHog) [inline code/game/gamemodes/gang/gang.dm]: Remove it, it is not as if it is going to make a significant difference in a Wizard round.

Comment (RemieRichards) [inline code/game/gamemodes/gang/gang.dm]: or give ```/datum/gang``` a ```is_deconvertible``` var, defaulting to true that just so happens to be false on the multiverse one. cmon @KorPhaeron think generic here.

File: code/modules/admin/verbs/onlyone.dm [hunk 3]
```diff
@@ -66,6 +66,13 @@
 		H << "<B>You are the multiverse summoner. Activate your blade to summon copies of yourself from another universe to fight by your side.</B>"
 		H.mind.announce_objectives()
 
+		var/datum/gang/multiverse/G = new(src, "[H.real_name]")
+		ticker.mode.gangs += G
+		G.fighting_style = "multiverse"
```
Comment (RemieRichards) [inline code/modules/admin/verbs/onlyone.dm]: this is already set in the definition.

File: code/game/gamemodes/gang/gang.dm (hunk 2 above)
Comment (KorPhaeron) [inline code/game/gamemodes/gang/gang.dm]: that sounds good yeah

**Comment by KorPhaeron** [comment]
I just remembered how dumb this is when combined with summon events/magic/guns

**Comment by Thunder12345** [comment]
Make it disable all other summons?

**Comment by KorPhaeron** [comment]
Also if I ever finish this: being ganged stuns you, I need to pass the proper argument so it won't in this case

**Comment by GunHog** [comment]
:'(

**Comment by KorPhaeron** [comment]
@optimumtact 

Done

**Comment by JJRcop** [comment]
image

**Comment by KorPhaeron** [comment]
@optimumtact 

For real now it's done

**Pull request merged**